cmake/InstallMKL.cmake

```diff
@@ -0,0 +1,109 @@
+#   Download MKL from Intel web site.
+
+message(STATUS "Downloading Intel MKL, this may take a while...")
+
+if (WIN32)
+    set(MKL_URL_DIR 12079)
+    set(MKL_BASE_NAME "w_mkl_2018.0.124")
+    set(MKL_INSTALLER_ARCHIVE "${MKL_BASE_NAME}.exe")
+elseif (APPLE)
+    # 2018.0 does not unsupport the MacOS version of travis...
+    #set(MKL_URL_DIR 12025)
+    #set(MKL_BASE_NAME "m_mkl_2018.0.104")
+    set(MKL_URL_DIR 12185)
+    set(MKL_BASE_NAME "m_mkl_2017.4.220")
+    set(MKL_INSTALLER_ARCHIVE "${MKL_BASE_NAME}.dmg")
+else()
+    set(MKL_URL_DIR 12070)
+    set(MKL_BASE_NAME "l_mkl_2018.0.128")
+    set(MKL_INSTALLER_ARCHIVE "${MKL_BASE_NAME}.tgz")
+endif()
+
+set(MKL_BASE_URL "http://registrationcenter-download.intel.com/akdlm/irc_nas/tec")
+file(DOWNLOAD "${MKL_BASE_URL}/${MKL_URL_DIR}/${MKL_INSTALLER_ARCHIVE}" ${CMAKE_BINARY_DIR}/${MKL_INSTALLER_ARCHIVE}
```

Comment:
it's here. Use SHOW_PROGRESS option cf:

https://cmake.org/cmake/help/v3.0/command/file.html